tests/ui/type-alias-impl-trait/ice-failed-to-resolve-instance-for-110696.rs

```diff
@@ -0,0 +1,52 @@
+// test for #110696
+// failed to resolve instance for <Scope<()> as MyIndex<()>>::my_index
+// ignore-tidy-linelength
```

Comment:
instead of that adapt the formatting/don't use the full message for the expected error? 🤔